projects/CenterPoint/configs/t4dataset/Centerpoint/second_secfpn_2xb8_121m_x2.py

```diff
@@ -189,6 +189,37 @@
     ),
 )
 
+# Add evaluator configs
+evaluator_configs = dict(
+    evaluation_task=_base_.evaluation_task,
+    label_prefix="autoware",
+    max_distance=[121.0, 121.0, 121.0, 121.0, 121.0],
+    min_distance=[0.0, 0.0, 0.0, 0.0, 0.0],
+    min_point_numbers=0,
+    target_labels=_base_.class_names,
+)
```

Comment:
```suggestion
evaluator_configs = dict(
    **_base_.evaluator_metric_configs,
    label_prefix="autoware",
    max_distance=121,
    min_distance=0,
    min_point_numbers=0,
)
```

I think we can include evaluator_metric_configs directly in evaluator_configs. This way, we wouldn’t need to pass evaluator_metric_configs separately into T4metrics, as they would already be combined at this stage.

Reply:
Additionally, max_distance and min_distance needs to be a float instead of a list :)

Reply:
Thanks! Updated